sv.h

```diff
@@ -855,8 +857,8 @@ Set the value of the MAGIC pointer in C<sv> to val.  See C<L</SvIV_set>>.
 Set the value of the STASH pointer in C<sv> to val.  See C<L</SvIV_set>>.
 
 =for apidoc Am|void|SvCUR_set|SV* sv|STRLEN len
-Set the current length of the string which is in the SV.  See C<L</SvCUR>>
-and C<SvIV_set>>.
+Sets the current length, in bytes, of the C string which is in the SV.
```

Comment:
I don't understands why this has to be a C string.  The SV may contain embedded NULs, and those are counted as bytes.

Reply:
Perhaps an explicit mention that the string may contain NULs and they are counted.

Reply:
Yeah I meant “C string” as distinct from “Perl string”, i.e., what `sv_len_utf8` does. When I looked at it I wasn’t sure if it referred to Perl `length()` or the number of bytes in the PV.

Maybe:
> Sets the current length of the PV inside the SV.

?

Reply:
Sets the current length, in bytes, of the PV inside the SV

would be better IMO

Reply:
Done.